resources/lang/en/general_content.php

```diff
@@ -142,7 +142,7 @@
     'label_trans_key'                          => 'Label',
     'customer_trans_key'                       => 'Customer',
     'status_trans_key'                         => 'Status',
-    'statu_trans_key'                          => 'Statu',
+    'statu_trans_key'                          => 'Status',
```

Comment:
I cannot accept this line, there is status and status with an "s" for the plural (line just before) if I maintain, it is duplicates

Reply:
Hm, But I guess there is no Statu in English, What can we do with this?

Reply:
I think we can leave the discussion for future. This line is reverted, please take another look.

Reply:
state or progress would be more appropriate? I validate the pull request and we will see how it is used later,

Reply:
I guess state and status are pretty similar. The only issue is that they are not related like it is in French. 

I can totally understand the meaning of Statu in UI, even though it's not English. Let's keep it as it is.